src/EnergyPlus/api/datatransfer.cc

```diff
@@ -573,6 +582,50 @@ Real64 getInternalVariableValue(EnergyPlusState state, int handle)
     return 0;
 }
 
+int getEMSGlobalVariableHandle(EnergyPlusState state, const char *name)
+{
+    auto *thisState = static_cast<EnergyPlus::EnergyPlusData *>(state);
+    int index = 0;
+    for (auto const &erlVar : thisState->dataRuntimeLang->ErlVariable) {
```

Comment:
Are you really sure this is the right array, at least unbounded whatsoever like this? More on this in a bit

Reply:
Great catch, so now I am skipping the built-in EMS variables.  If someone wants to get one of those built-in variables, they can get them through normal API methods.